playwright/_impl/_frame.py

```diff
@@ -181,6 +181,21 @@ async def continuation() -> Optional[Response]:
 
         return EventContextManagerImpl(asyncio.create_task(continuation()))
 
+    async def wait_for_url(
+        self,
+        url: URLMatch,
+        wait_until: DocumentLoadState = None,
+        timeout: float = None,
+    ) -> None:
+        matcher = URLMatcher(url)
+        if matcher.matches(self.url):
+            await self.wait_for_load_state(state=wait_until, timeout=timeout)
+            return
+        async with self.expect_navigation(
```

Comment:
very odd alignment :/

Reply:
it isn't me though.

Reply:
joke of the day: _seems like [black](https://github.com/psf/black) magic_